lib/ansible/modules/windows/win_uri.ps1

```diff
@@ -264,8 +264,11 @@ if ($return_content -or $dest) {
     }
 }
 
-if ($status_code -notcontains $response.StatusCode) {
-    Fail-Json -obj $result -message "Status code of request '$($response.StatusCode)' is not in list of valid status codes $status_code."
+# use numerical format of response status for comparison
+$resp_code = [System.Enum]::Format([System.Net.HttpStatusCode], $response.StatusCode, "d")
+
+if ($status_code -notcontains $resp_code) {
+    Fail-Json -obj $result -message "Status code of request '$resp_code' is not in list of valid status codes $status_code."
```

Comment:
Having the int value there is great (and correct) but it would be nice to also include the text description of the code next to it like before. I believe this is how the Python module does it as well.